airflow/providers/google/cloud/utils/credentials_provider.py

```diff
@@ -201,7 +201,7 @@ def __init__(
         key_options = [key_path, key_secret_name, keyfile_dict]
         if len([x for x in key_options if x]) > 1:
             raise AirflowException(
-                "The `keyfile_dict`, `key_path`, and `key_secret_name` fields"
+                "The `keyfile_dict`, `key_path`, and `key_secret_name` fields "
                 "are all mutually exclusive. Please provide only one value."
```

Comment:
Wow I’m impressed this is being actively caught.

Reply:
I'd be impressed too but I found it fixing some concatenation in the `test_base_google.py` file. Which means we had to fix the code since the test would fail. 🙃